src/CountryListService.php

```diff
@@ -4,18 +4,11 @@
 
 class CountryListService
 {
-    /**
-     * @return array
-     */
     public function blacklistedCountries(): array
     {
-        return ['AL','BY','BA','BI','CF','HR','HU','CU','KP','CD','IR','IQ','QZ','LA','LB','LY','MK','ME','NI','RS','SO','SS','SD','SY','UA','VE','YE','ZW','BS','BB','BJ','BF','KH','KY','TD','CO','KM','CG','DO','GQ','ER','GN','GW','HT','LS','LR','MG','MM','PA','UG','PR'];
+        return ['AL','BY','BA','BI','CF', 'CU','KP','CD','IR','IQ','QZ','LA','LB','LY','MK','ME','NI','RS','SO','SS','SD','SY','UA','VE','YE','ZW','BS','BB','BJ','BF','KH','KY','TD', 'KM','CG','DO','GQ','ER','GN','GW','HT','LS','LR','MG','MM','PA','UG','PR'];
```

Comment:
We need to make sure what limitations any new countries that were added are (outside of HU, HR, CO which we know need to be limited to neocurreny - egiftcards only)

Reply:
Looks like Kim said if they weren't on blacklist and new, they should be treated like all other countries. Makes sense, cause if they were bad, they would've been on blacklist. D'oh. Resolving this comment.